docs/chaos-engineering/chaos-faults/aws/ec2-io-stress.md

```diff
@@ -0,0 +1,312 @@
+kind: ChaosEngine
+metadata:
+  name: engine-nginx
+spec:
+  engineState: "active"
+  annotationCheck: "false"
+  chaosServiceAccount: ec2-io-stress-sa
+  experiments:
+  - name: ec2-io-stress
+    spec:
+      components:
+        env:
+        - name: FILESYSTEM_UTILIZATION_PERCENTAGE
+          VALUE: '50'
+        # id of the ec2 instance
+        - name: EC2_INSTANCE_ID
+          value: 'instance-1'
+        # region for the ec2 instance
+        - name: REGION
+          value: 'us-east-1'
+```
+
+### MULTIPLE WORKERS
+
+It defines the CPU threads to be run to spike the filesystem utilisation, this will increase the growth of filesystem consumption. It can be tuned via `NUMBER_OF_WORKERS` ENV.
+
+Use the following example to tune this:
+
+[embedmd]:# (./static/manifests/ec2-io-stress/multiple-workers.yaml yaml)
+```yaml
+# multiple workers to utilize resources
+apiVersion: litmuschaos.io/v1alpha1
+kind: ChaosEngine
+metadata:
+  name: engine-nginx
+spec:
+  engineState: "active"
+  annotationCheck: "false"
+  chaosServiceAccount: ec2-io-stress-sa
+  experiments:
+  - name: ec2-io-stress
+    spec:
+      components:
+        env:
+        - name: NUMBER_OF_WORKERS
+          VALUE: '3'
+        # id of the ec2 instance
+        - name: EC2_INSTANCE_ID
+          value: 'instance-1'
+        # region for the ec2 instance
+        - name: REGION
+          value: 'us-east-1'
+```
+
+### VOLUME MOUNT PATH
+
+It defines volume mount path to target attached to the EC2 instance. It can be tuned via `VOLUME_MOUNT_PATH` ENV.
+
+Use the following example to tune this:
+
+[embedmd]:# (./static/manifests/ec2-io-stress/volume-path.yaml yaml)
+```yaml
+# volume path to be used for io stress
+apiVersion: litmuschaos.io/v1alpha1
+kind: ChaosEngine
+metadata:
+  name: engine-nginx
+spec:
+  engineState: "active"
+  annotationCheck: "false"
+  chaosServiceAccount: ec2-io-stress-sa
+  experiments:
+  - name: ec2-io-stress
+    spec:
+      components:
+        env:
+        - name: VOLUME_MOUNT_PATH
+          VALUE: '/tmp'
+        # id of the ec2 instance
+        - name: EC2_INSTANCE_ID
+          value: 'instance-1'
+        # region for the ec2 instance
+        - name: REGION
+          value: 'us-east-1'
+```
+
+### MULTIPLE EC2 INSTANCES
+
+Multiple EC2 instances can be targeted in one chaos run. It can be tuned via `EC2_INSTANCE_ID` ENV.
+
+Use the following example to tune this:
+
+[embedmd]:# (./static/manifests/ec2-io-stress/multiple-instances.yaml yaml)
+```yaml
+# mutilple instance targets
+apiVersion: litmuschaos.io/v1alpha1
+kind: ChaosEngine
+metadata:
+  name: engine-nginx
+spec:
+  engineState: "active"
+  annotationCheck: "false"
+  chaosServiceAccount: ec2-io-stress-sa
+  experiments:
+  - name: ec2-io-stress
+    spec:
+      components:
+        env:
+        - name: MEMORY_CONSUMPTION
+          VALUE: '1024'
+        # ids of the ec2 instances
+        - name: EC2_INSTANCE_ID
+          value: 'instance-1'
```

Comment:
```suggestion
          value: 'instance-1,instance-2'
```

Reply:
Have provided multiple values in the multiple-instances.yaml example.